src/Language/Mulang/Inspector/Generic/Smell.hs

```diff
@@ -1,7 +1,10 @@
+{-# LANGUAGE ViewPatterns #-}
+
 module Language.Mulang.Inspector.Generic.Smell (
   hasRedundantBooleanComparison,
   hasRedundantIf,
   hasRedundantGuards,
+  shouldUseOtherwise,
```

Comment:
This doesn't follow the rest of the smells' naming scheme. Feel free to disregard

Reply:
You are right!